doc/GUIDE.md

```diff
@@ -103,17 +144,6 @@ In the following section we'll go over examples of declarations that might be fo
 header file, show the code that jextract generates for these declarations, and show
 some examples of how to use the generated Java code.
 
-Most of the code that jextract generates will be available through a single class. By default
-the name of that class is derived from the name of the main header file that is passed to
-jextract. For example, if the header file is named `mylib.h`, then the derived class name
-will be `mylib_h`. The class name can be set explicitly using the `--header-class-name`
-option as well.
-
-Besides the main header class that is generated, jextract also generates several
-separate files for certain declarations in the C header file. Namely, structs, unions,
-function pointer types, and typedefs of struct or union types result in additional files
-being generated. We will look at those in more detail in this section.
-
 Most of the methods that jextract generates are `static`, and are designed to be imported
```

Comment:
Uhm - I think the paras you moved fitted nicely in this introductory section

Reply:
I think they could go in either section? But, this way, we can discuss the `--header-class-name` option in the section about running, which I think makes more sense.

Reply:
On a second look, it makes sense in this updated version.